src/app/server/Server.h

```diff
@@ -479,6 +504,19 @@ class Server
             }
         }
 
+        void ClearSubscriptionResumptionStateOnFabricChange(chip::FabricIndex fabricIndex)
```

Comment:
This is really fabric removal, not fabric change...  For CASE it's fabric change because UpdateNOC needs to do the CASE things too, but this bit is removal-specific.